### DISCUSSION TITLE
Host v3 prerelease packages in NuGet

### DISCUSSION THREAD

**Discussion by TylerBrinkley** [xunit/xunit#2759] [General]
With the MyGet outage still occurring for the past 24 hours would you consider moving the v3 prerelease packages to NuGet.org?

**Comment by bradwilson** [comment]
I haven't done it yet because I feel like in its current state it's too soon due to the lack of ability to run tests in third party runners (especially Test Explorer in Visual Studio).

Our CI builds on GitHub keep the packages around and downloadable, so it would be trivial for me to push them somewhere else. I haven't really decided where that might be yet; I'm looking into options (and hoping the MyGet team gets the servers back online in the meantime). One that I considered but discarded is GitHub Packages because it requires people to be authenticated to pull packages (no anonymous support), but that may end up still being the best option even so. We'll see.
